cpe_crypt/README.md

```diff
@@ -45,18 +49,18 @@ The profile delivers a payload for the above keys in `node['cpe_crypt']`.  The t
 For example, you could tweak the above values
 
     node.default['cpe_crypt']['install'] = true
-    node.default['cpe_crypt']['preferences']['ServerURL'] =
+    node.default['cpe_crypt']['prefs']['ServerURL'] =
       'https://crypt.somewhere.com'
-    node.default['cpe_crypt']['preferences']['SkipUsers'] = [
+    node.default['cpe_crypt']['prefs']['SkipUsers'] = [
       'supersecretadmin',
       'noreallysupersecretadmin'
     ]
-    node.default['cpe_crypt']['preferences']['RemovePlist'] = false
-    node.default['cpe_crypt']['preferences']['RotateUsedKey'] = false
-    node.default['cpe_crypt']['package'] = {
+    node.default['cpe_crypt']['prefs']['RemovePlist'] = false
+    node.default['cpe_crypt']['prefs']['RotateUsedKey'] = false
+    node.default['cpe_crypt']['pkg'] = {
       'app' => 'crypt',
       'checksum' =>
         '1582e974820a5b27cfe462521cb0d4802319224753f3d6417ee23fff9333872a',
-      'receipt' => 'com.grahamgilbert.Crypt',
+      'receipt' => 'com.grahamgilbert.crypt',
       'version' => '3.0.0.109',
```

Comment:
I am not sure I agree with this case change. Maybe fb is rebuilding crypt? However the default receipt has a capital 'C' and `pkgutil` is case sensitive. 

```bash
$ pkgutil --pkg-info com.grahamgilbert.Crypt
package-id: com.grahamgilbert.Crypt
version: 3.0.0.109
volume: /
location: /
install-time: 1507948451

$ pkgutil --pkg-info com.grahamgilbert.crypt
No receipt for 'com.grahamgilbert.crypt' found at '/'.
```

Reply:
I agree. I will merge this, but fix.